### DISCUSSION TITLE
I can't build my page with NextJS + Vercel

### DISCUSSION THREAD

**Discussion by lucasandradegs** [vercel/next.js#60746] [Help] [answered]
### Summary

I'm doing my first landing page with NextJS and Vercel, I finished the mobile version and I would like to deploy the project to see what looks like. When I try to build the repository on Vercel, I get this error every time:

**Type error: Page "src/app/page.jsx" does not match the required types of a Next.js Page.
"Container" is not a valid Page export field.**

My repository: https://github.com/lucasandradegs/desafio-prominas

*locale, using npm run dev it's normal.

**Comment by Amir-R1384** [comment]
In your app/page file, try wrapping your container around an empty div

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The build step in App Router, does a strict check on what is exported from a `page.tsx` or `page.jsx`, or `page.js`, file. 


In your case, you are exporting Container, but that is not allowed. You can have a:

- `default` export, 
- you can export `generateStaticParams`, 
- you can export `metadata` or `generateMetadata`
- or any of the route segment config options, https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config
- and I think there might be a few others

However, Container is not one of them, so do this:

```diff
- export const Container = styled.div`
+ const Container = styled.div`
```

The same applies to `Content`, `Section`, `SecondSection`, etc... it is better to move these to other folders, for example `src/components`

**Reply by lucasandradegs** [reply]
Yeah, this was the problem. It was my first time using nextjs and styled-component, so I didn`t know if I could do the style on a different page, every video I saw it was nextjs + tailwind. Thank you!
